js/router.js

```diff
@@ -48,6 +48,10 @@ module.exports = Backbone.Router.extend({
         this.navigate(translatedRoute, { trigger: true });
       });
     });
+    
+    history.size = -1;
```

Comment:
On another note, hard-coding the size and position here assumes that on app start the history is empty, which is a wrong assumption. The browser maintains the history on app reload.

In other words, before this PR, let's say the user installed the app for the first time, so they have a fresh history. Now they navigate to 15 different pages. If they restart the app, they will be able to hit Back 14 times to see their previous 14 pages. After this PR, you are completely ignoring that existing history and those previous 14 entries in the history queue are ignored and blocked away from the user.